src/config.yml

```diff
@@ -213,14 +213,20 @@ announcer:
     - 'Be sure to visit our forums at &6http://totalfreedom.boards.net/'
     - 'You can always review the server rules here: &6http://totalfreedom.me/'
     - 'If you are not OP, be sure to ask!'
+    - 'Players with their prefix/tag as '[Helper]' aren't staff and it isn't a rank, you can set your own tag by doing /tag set <tag>'
     - 'Somebody breaking the rules?  Report it!  /report <user> <reason>'
     - 'Griefing is not allowed!'
+    - 'Spamming is not allowed!'
+    - '&4TotalFreedom &b- Giving players joy since 2010!'
+    - '"Logic is just an illusion, don't expect to benefit from logic, because it doesn't exist in this world"'
```

Comment:
Not sure anyone is going to even attempt to understand this one... Possibly better to stick to clean, concise and just what links directly for TF